docs/general/about-us.md

```diff
@@ -80,6 +81,12 @@ The team responsible to keep the this community a healthy place (at least for th
 - [Suffa28](https://retroachievements.org/user/Suffa28)
 - [WilHiteWarrior](https://retroachievements.org/user/WilHiteWarrior)
 
+### Community Manager
+
+The Community Manager helps facilitate communication between players and staff, navigate misunderstandings and conflicts, oversee our various social platforms, and moderation team support:
```

Comment:
```suggestion
The Community Manager helps facilitate communication between players and staff, navigates misunderstandings and conflicts, oversees our various social platforms, and provides moderation team support:
```

Reply:
I wrote this differently than how you and the chatbot I just checked it with read it.  In my mind it was the CM **helps** <facilitate comm...>, <navigate mis...>, <oversee our...>  not the CM <helps facili>, <navigates mis>...

Definitely missed the the "provide(s)" though.

All changes commited as suggested.